src/operator/tensor/elemwise_binary_op_basic.cc

```diff
@@ -57,7 +58,12 @@ NNVM_REGISTER_OP(_backward_add)
 .set_attr<FComputeEx>("FComputeEx<cpu>",
                       ElemwiseBinaryOp::BackwardUseNoneEx<cpu, mshadow_op::identity,
                       mshadow_op::identity>)
-.set_attr<FInferStorageType>("FInferStorageType", ElemwiseStorageType<1, 2, true, true, false>);
+.set_attr<FInferStorageType>("FInferStorageType",
+                             ElemwiseStorageType<1, 2, true, true, false>)
```

Comment:
Could you change it to ElemwiseStorageType<1, 2, true, true, true> since CSR tests are added now?

Reply:
done